imports/ui/pages/questions_library.jsx

```diff
@@ -389,69 +391,49 @@ export const QuestionsLibrary = createContainer(props => {
   const handle =  Meteor.subscribe(subscription, props.courseId)
   const courseId = props.courseId
   
-  
   let params = {}
   let editable = true
   
   if (props.library === 'library') {
```

Comment:
I think that everything that you are adding to the query is already handled in the publication/subscription. If not, it should be, right?